src/code/main.c

```diff
@@ -52,8 +73,20 @@ void Main(void* arg) {
     gAppNmiBufferPtr = (PreNmiBuff*)osAppNMIBuffer;
     PreNmiBuff_Init(gAppNmiBufferPtr);
     Fault_Init();
+#if PLATFORM_N64
+    func_800ADA80();
+    if ((u8)B_80121AE1 != 0) {
+        systemHeapStart = (uintptr_t)&D_801E8090;
+        SysCfb_Init(1);
+    } else {
+        func_800ADAF8();
+        systemHeapStart = (uintptr_t)_buffersSegmentEnd; // D_801C7720_unknown
```

Comment:
remove comment?

Reply:
Is `_buffersSegmentEnd` right?

Reply:
Yeah I agree with that. It seems like buffers comes just before n64dd (which starts at 801C7740), so I think that's wrong in the spec currently. And D_801E8090 is _n64ddSegmentEnd

I was thinking that either this code is right (in which case we may forget to remove the comment later) or the code isn't (in which case we'll have to revisit it later anyway) so it seems better to remove the comment